src/chatbot/src/chatbot/aiml/DefaultSubs.py

```diff
@@ -48,8 +50,8 @@
     "hers": "his",
     "herself": "himself",
 }
-
-defaultPerson = {
+# These now match the comment above
+defaultPerson2 = {

```

Comment:
great.

On Sat, Aug 20, 2016 at 12:09 PM, Linas Vepštas <notifications@github.com>
wrote:

> In src/chatbot/src/chatbot/aiml/DefaultSubs.py
> <https://github.com/hansonrobotics/HEAD/pull/168#discussion_r75581851>:
>
> > @@ -48,8 +50,8 @@
> >      "hers": "his",
> >      "herself": "himself",
> >  }
> > -
> > -defaultPerson = {
> > +# These now match the comment above
> > +defaultPerson2 = {
>
> Yikes!
> https://www.chatbots.org/ai_zone/viewthread/1839/
>
> seems to be saying that the definition of person and person2 was flipped
> between AIML version 1.0 and 2.0 and there is assorted confusion as a
> result.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/hansonrobotics/HEAD/pull/168/files/85ee952bbb4ef504d4b96fb53f07abcfdd7700a7#r75581851>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF3yl1Kt-RZkz4Zlc-86b9jhF2e2Z4uQks5qhzSugaJpZM4Jo5ec>
> .
>